omero/sysadmins/unix/install-web.rst

```diff
@@ -1,39 +1,12 @@
-Setting up OMERO.web
-====================
+OMERO.web administration
```

Comment:
From the filename, URL and location, this is effectively the install web index page, which makes co-opting it into the admin page slightly confusing if you're using the 'previous' and 'next topic's in the sidebar to navigate rather than just the main index page.

I'm not sure there is really a way to fix that without breaking some URLs though...

Reply:
That's the problem I have with the re-organisation
I have tried to change things w/o url break